magenta/interfaces/midi/midi.py

```diff
@@ -510,6 +539,9 @@ def main(unused_argv):
           '--stop_capture_control_value must both be defined and unique.')
     return
 
+  if not 0 <= FLAGS.metronome_playback_velocity <= 127:
+    raise ValueError('The velocity must be an integer between 0 and 127')
```

Comment:
Let's actually do a print and return like we do above since this is the main function.